packages/dbml-parse/src/lib/interpreter/types.ts

```diff
@@ -18,6 +18,8 @@ export interface InterpreterDatabase {
   refIds: { [refid: string]: ElementDeclarationNode };
   ref: Map<ElementDeclarationNode, Ref>;
   enums: Map<ElementDeclarationNode, Enum>;
+  // for keeping track of the owner table group of a table
+  tableOwnerGroup: { [tableid: string]: ElementDeclarationNode };
```

Comment:
The current name is fine but I suggest the following names:
- groupOfTable
- groupNodeOfTable

Reply:
Renamed to `groupOfTable`!